pkg/cmd/cluster/config_wrapper.go

```diff
@@ -50,6 +50,10 @@ type configWrapper struct {
 	clusterObj    *appsv1alpha1.Cluster
 	clusterDefObj *appsv1alpha1.ClusterDefinition
 	clusterVerObj *appsv1alpha1.ClusterVersion
+
+	// new API
```

Comment:
It is best to specify which version of KB introduced the API. Either don't write this comment, or make it clear.